tools/release.js

```diff
@@ -109,7 +108,7 @@ const Prompts = {
       await this.askQuestion(promptOps);
     } catch (err) {
       const warning = colors.yellow(
-        "🚨 Make sure to run `git up; git checkout develop`"
+        "🚨 Make sure to run `git up; git checkout main`"
```

Comment:
What's this `git up` here?

Reply:
oh yeah, lol... `git up`... that super ol' school, lol. I think it did a `git pull` + rebase or something 

I think it was this... 
https://github.com/git-up/GitUp